src/Platform/Microsoft.Testing.Platform/OutputDevice/Terminal/TerminalTestReporter.cs

```diff
@@ -887,7 +887,7 @@ public void AppendTestDiscoverySummary(ITerminal terminal)
     {
         terminal.AppendLine();
 
-        var assemblies = _assemblies.Select(asm => asm.Value).OrderBy(a => a.Assembly).Where(a => a is not null).ToList();
+        var assemblies = _assemblies.Values.OrderBy(a => a.Assembly).ToList();
```

Comment:
@nohwnd We only ever have one assembly right? I think it might be worth if we clean this up.